src/nvim/api/vim.c

```diff
@@ -1562,3 +1563,18 @@ Object nvim_get_proc(Integer pid, Error *err)
 #endif
   return rvobj;
 }
+
+/// Give the info of the state of the Capslock, Numlock and Scrollock
+///
+/// @return the dictionary holding the data about the locks
```

Comment:
This is not descriptive. You need to specify what keys are possible exactly, what are corresponding values (booleans) and under which circumstances they appear (i.e. state that if lock is checked it will appear, if it is not then it won’t).

Reply:
OK, sure